test/a11y.html

```diff
@@ -47,13 +47,15 @@
 
         test('aria attributes', function() {
           assert.equal(collapse.getAttribute('role'), 'group');
-          assert.equal(collapse.getAttribute('aria-expanded'), 'false');
           assert.equal(collapse.getAttribute('aria-hidden'), 'true');
+          // aria-expanded should only be set on the button that
+          // controls the collapsable area, not on the area itself.
+          // See https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element#Description
+          assert.isNotOk(collapse.getAttribute('aria-expanded'));
```

Comment:
nit: I'd use `assert.isFalse(collapse.hasAttribute('aria-expanded'))` here, to also verify that we don't set the attribute itself.